package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "kiloview-encoder",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Please bump to 2.0.0
https://github.com/bitfocus/companion/wiki/Versioning

Reply:
ok version bumped and thanks for the docs, seems i missed that part